manifest.webapp

```diff
@@ -14,11 +14,26 @@
   },
   "default_locale": "en",
   "locales": {},
+  "intents": [
+    {
+      "action": "INSTALL",
+      "data": ["slug"],
+      "href": "/intents",
+      "type": [
+        "io.cozy.konnectors"
```

Comment:
Should we use a generic `io.cozy.apps` like for the redirection? I mean we could need to install an application from the intent? Or is this completely different? I'd like to keep things generic between konnectors and webapps in the Store.